app/components/SessionExpiryHandler.tsx

```diff
@@ -62,10 +62,11 @@ const SessionExpiryHandler: React.FC = () => {
         <SessionTimeoutHandler
           modalDisplaySecondsBeforeLogout={120}
           onSessionExpired={handleSessionExpired}
+          resetOnChange={[router]}
```

Comment:
This listens for next router changes and refreshes the session on each router change. This *should* severely mitigate the invalid session modal popup.